lib/Dialect/EmitC/Conversion/TosaToEmitC.cpp

```diff
@@ -117,6 +117,36 @@ class FullyConnectedOpConversion
   }
 };
 
+class MatMulOpConversion : public OpConversionPattern<tosa::MatMulOp> {
+  using OpConversionPattern<tosa::MatMulOp>::OpConversionPattern;
+
+public:
+  MatMulOpConversion(MLIRContext *ctx, StringRef funcName)
```

Comment:
```suggestion
  MatMulOpConversion(MLIRContext *ctx)
```
No need to pass `funcName`, you hardcode it anyway.